Luciana Godio/style.css

```diff
@@ -0,0 +1,216 @@
+nav{
+    display: flex;
+    align-items: center;
+    background-color: black;
+    width: 100vw;
+    height: 60px;
+    margin: -9px 0 0 -9px;
+    position: sticky;
+    z-index: 1;
+    
+}
+.nav-lista{
```

Comment:
suggestion: order by tags, classes and ids in this order to make easier to read the file